Spec/RestPaginated.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  RestPaginated.swift
+//  Ably
+//
+//  Created by Ricardo Pereira on 23/08/2018.
+//  Copyright © 2018 Ably. All rights reserved.
+//
+
+import Ably
+import Nimble
+import Quick
+import SwiftyJSON
+
+class RestPaginated : QuickSpec {
+    override func spec() {
+        describe("RestPaginated") {
+
+            let links = "<./messages?start=0&end=1535035746063&limit=100&direction=backwards&format=msgpack&firstEnd=1535035746063&fromDate=1535035746063&mode=all>; rel=\"first\", <./messages?start=0&end=1535035746063&limit=100&direction=backwards&format=msgpack&firstEnd=1535035746063&fromDate=1535035746063&mode=all>; rel=\"current\""
+
+            let url = URL(string: "https://sandbox-rest.ably.io:443/channels/foo/messages?limit=100&direction=backwards")!
```

Comment:
This won't work as soon as we change the value of `ABLY_ENV`

Reply:
Those requests will fail so I think they doesn't need look at ABLY_ENV. I just use `sandbox` to not overload a production server.